app/submitter/submitter.py

```diff
@@ -23,18 +23,18 @@ def __init__(self, project_id, bucket_name):
         client = storage.Client(project_id)
         self.bucket = client.get_bucket(bucket_name)
 
-    def send_message(self, message, case_id=None, tx_id=None):
+    def send_message(self, message, tx_id=None, case_id=None):
```

Comment:
Are these ever `None`?

Reply:
Both are defined as required
https://ons-schema-definitions.readthedocs.io/en/latest/respondent_management_to_electronic_questionnaire.html#required-fields

Reply:
RM would be unable to receipt with a message containing no `case_id` in the meta, so would appreciate this as required.

Reply:
I've removed code to check this as the app already make it mandatory hence would fail early if the values are not there.